multiversx_sdk_cli/constants.py

```diff
@@ -6,3 +6,5 @@
 DEFAULT_CARGO_TARGET_DIR_NAME = "default_cargo_target"
 TRANSACTION_OPTIONS_TX_GUARDED = 0b0010
 ONE_YEAR_IN_SECONDS = 60 * 60 * 24 * 365
+DEFAULT_HRP = "erd"
+ADDRESS_ZERO = "erd1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6gq4hu"
```

Comment:
might rename it to `ADDRESS_ZERO_BECH32` ?

Reply:
Renamed